### PR TITLE
[Feature] Add support for env files

### PR DESCRIPTION
**Pull request by jackwrfuller**
Environment variable files (i.e ".env") are a commonly used method to define and manage project specific secrets and configuration.

It is standard for yaml-like projects to allow you to access env vars you have defined in env files in your project. For example, Docker Compose will read variables defined in an ".env" file by default without any configuration.

Currently, to access a variable `FOO=bar` defined in a file ".env" in a command, you would need prepend a `source .env` to the command each and every time:

```bash
source .env && echo $FOO
```

This is tiresome and error-prone. In my proposed PR, we would allow the user to define _global_ and _command-level_ env files. For example:

```yaml
ahoyapi: v2
env: .env
commands:
  bar:
    cmd: echo $BAR
  foo:
    cmd: echo $FOO
    env: .env.default
```
In the case where a command has both the global and command-level env files set, and these files contain conflicting variable definitions, I propose that the command-level env file overrides the global one.

Also, as it is currently implemented the filepath provided to the `env:` yaml key must be relative to the Ahoy source directory; see for example:

```Go
if config.Env != "" {
        globalEnvFile := filepath.Join(AhoyConf.srcDir, config.Env)
	envVars = append(envVars, getEnvironmentVars(globalEnvFile)...)
}
```

### CODE REVIEW

**Comment by ocean** [comment]
Thanks for these two PRs @jackwrfuller 🙌

I'll have a look in the next couple of days and make a new release.

File: testdata/.env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+GLOBAL=global
```
Comment (AlexSkrypnyk) [inline testdata/.env:1]: please include comments in this file

File: ahoy.go [hunk 2]
```diff
@@ -172,8 +174,38 @@ func getSubCommands(includes []string) []cli.Command {
...
+		envVars = append(envVars, line)
+	}
+	return envVars
+}
+
 func getCommands(config Config) []cli.Command {
 	exportCmds := []cli.Command{}
+	envVars := []string{}
+
+	// If a global environment variable file is defined, use that too.
```
Comment (AlexSkrypnyk) [inline ahoy.go:204]: please clarify what "use that too" means exactly: will the values in the global file override the values in the local file or the other way around?

**Review by AlexSkrypnyk — CHANGES_REQUESTED** [review]

File: testdata/.env (hunk 1 above)
Comment (jackwrfuller) [reply testdata/.env:3]: I have added comments clarifying the intention of both variables in this file.

File: ahoy.go (hunk 2 above)
Comment (jackwrfuller) [reply ahoy.go]: The other way around: 'global' env files are overridden by command-level ones. 

The "use that too" is actually leftover from my original implementation, where Ahoy would read a `.env` file by default if it was present, like what docker compose currently does.

**Pull request merged**